### PR TITLE
Update frontend with navbar and sidebar

### PR DESCRIPTION
**Pull request by ansonjwhe**

### CODE REVIEW

File: frontend/src/components/Dashboard.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useReducer, useState } from 'react'
+import { useAuth } from '../contexts/AuthContext'
+import axios from 'axios'
+import { useLocation, useNavigate } from 'react-router-dom'
+import TicketList from './TicketList'
+import Navbar from './Navbar'
+import { Text } from '@chakra-ui/react'
+
+export const SF_DATA_KEY = 'sfData'
```
Comment (victorzheng02) [inline frontend/src/components/Dashboard.js]: Put these all in an Object.freeze just so we can group them together and simulate an enum

File: frontend/src/components/Login.js [hunk 2]
```diff
@@ -1,31 +1,68 @@
...
+} from '@chakra-ui/react'
 
 const Login = () => {
     const [error, setError] = useState('')
     const [loading, setLoading] = useState(false)
     const { login } = useAuth()
     const navigate = useNavigate()
+    const prefersReducedMotion = usePrefersReducedMotion()
+
+    const bgKeyframe = keyframes`
```
Comment (victorzheng02) [inline frontend/src/components/Login.js]: anyway to remove this bundle of keyframes outside the component? Maybe in a util file? I don't like seeing it live in the login component

File: frontend/src/components/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { Button, Flex, Heading, Spacer } from '@chakra-ui/react'
+import React from 'react'
+
+const Navbar = (props) => {
+    return (
+        <Flex alignItems="center" p="16px 24px" bgColor="deepskyblue">
```
Comment (victorzheng02) [inline frontend/src/components/Navbar.js:9]: if we're going to start using specific colours like deepskyblue, lets get a styled-system going and define them so they can be more reusable
https://chakra-ui.com/docs/styled-system/theme

File: frontend/src/components/Login.js [hunk 4]
```diff
@@ -1,31 +1,68 @@
...
     return (
-        <div>
-            <button disabled={loading} onClick={handleLogin}>
-                Log In
-            </button>
-            {error && <span>{error}</span>}
-        </div>
+        <>
+            <Navbar
+                onClick={handleLogin}
```
Comment (victorzheng02) [inline frontend/src/components/Login.js:28]: I cannot login ever, automatically fails

**Review by victorzheng02 — COMMENTED** [review]
Great work, small nits and I want to discuss login before we merge

File: frontend/src/components/Login.js (hunk 4 above)
Comment (ansonjwhe) [reply frontend/src/components/Login.js:28]: This is caused by not having the Google OAuth sign-in method enabled in your Firebase Console. Readme has been updated to include instructions for how to set this up

File: frontend/src/components/Login.js (hunk 2 above)
Comment (ansonjwhe) [reply frontend/src/components/Login.js]: Opted to remove the animation and keep a simple gradient. We'll replace the gradient with some sort of a WATO background image in the future anyways

File: frontend/src/components/Navbar.js (hunk 3 above)
Comment (ansonjwhe) [reply frontend/src/components/Navbar.js:9]: [Create login page view #2](https://github.com/WATonomous/wato_finance_system/issues/2) This ticket should be completed first to define/standardize our colour palette and theme styles. "deepskyblue" is a close-enough colour that can be used temporarily until we get official designs.

**Review by victorzheng02 — APPROVED** [review]
Looks good now!

**Pull request merged**